uint8 lu_x
uint8 lu_y
rospix_classification/ClusterClass cluster_class
rospix_classification/Pixel[] pixels
